lib/grafana_annotations/annotation.rb

```diff
@@ -4,7 +4,7 @@ module GrafanaAnnotations
   class Annotation < Dry::Struct
     attribute? :dashboardUID, Types::Strict::String.optional
     attribute? :panelId, Types::Strict::String.optional
-    attribute :time, Types::Strict::Integer
+    attribute :time, Types::Strict::Integer.default { GrafanaAnnotations::Utils::Time.now_ms } # rubocop:disable Lint/AmbiguousBlockAssociation
```

Comment:
We should avoid suppressing this warning. It'd better to fix a warning by adding `()` around `#attribute` arguments.

Reply:
fixed